tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java

```diff
@@ -154,4 +167,58 @@ public void beginTask( String taskName, int totalWork )
         return null;
     }
 
+    private void clearExistMarkerFromProject( IProject project )
```

Comment:
Should be called "clearExistingMarkersFromResourcet" and move this to the MigrationUtil Class.